lib/Prelude.dbl

```diff
@@ -87,6 +87,18 @@ pub method makeString {`re : {type X} -> Unit ->[|_] X, self : String}
     (extern dbl_strMake : Int -> String) n
   else `re ()
 
+pub method toList     = (extern dbl_strToListChr : String -> List Char) self
+pub let charListToStr = (extern dbl_chrListToStr : List Char -> String)
```

Comment:
As above, this function is quite complex. Can we implement it using simpler means?

Reply:
```toList``` could be made using strings' ```get``` method and ```charListToStr``` using string concatenation,  but I am really worried about performance of said approach.

Reply:
In `toList` using `get` will be similar to the current implementation, but in different language. There should be no performance issues here.

As for `charListToStr`, using concatenation would have logarithmic performance overhead if implemented properly. Maybe it is not a big overhead, but for me it may justify to implement it as extern. However, if we add to the library mutable and freezable arrays (I talked about them with @Foxinio, and I'm going to add an issue that explain our idea) then we would be able to provide efficient and simple implementation of `charListToString`. As for now, I'm happy with current implementation.